### PR TITLE
App signal logging

### PR DESCRIPTION
**Pull request by ivanvelkov**

### CODE REVIEW

File: src/server/AppsignalLogger.js [hunk 1]
```diff
@@ -0,0 +1,73 @@
...
+		if (config.isProd) {
+			try {
+				this.logger = Appsignal.client.logger('Root', logLevel);
+			} catch {
+				console.log('Appsignal nodejs lib could not be started, falling back to http logging');
+			}
+		}
+	}
+
+	log(message, attributes) {
```
Comment (sjmueller) [inline src/server/AppsignalLogger.js:18]: Should `logger.log` be considered `level:debug`? In this codebase, all of the `console.log` would be well suited at `level:info`. 

Would hate to enable debug level in prod to get this info.

File: src/server/ClientHandle.js [hunk 2]
```diff
@@ -203,7 +203,7 @@ class ClientHandle {
 		// Kick player if hasn't responded from ping
 		if (this.pingStart != null && Date.now() - this.pingStart > config.pingTimeout * 1000) {
 			this.pingStart = null;
-			console.log(`Client timed out: ${this.id}`);
+			logger.log(`Client timed out: ${this.id}`);
```
Comment (sjmueller) [inline src/server/ClientHandle.js:206]: Maybe as simple as these `console.log` converts to `logger.info`?

File: src/server/CultServer.js [hunk 3]
```diff
@@ -164,14 +164,16 @@ class CultServer {
 	}
 
 	startLobby() {
-		console.log('Setting lobby ready');
+		logger.info('Setting lobby ready');
```
Comment (sjmueller) [inline src/server/CultServer.js:167]: Which I see is done for some (like this)

Comment (ivanvelkov) [reply src/server/CultServer.js:167]: Did not want to pollute the logs, feel free to change whichever log instance you need from .log to .info

**Pull request merged**